README.md

```diff
@@ -26,22 +28,20 @@ Parse Server works with the Express web application framework. It can be added t
     - [Running elsewhere](#running-parse-server-elsewhere)
         - [Sample Application](#parse-server-sample-application)
         - [Parse Server + Express](#parse-server--express)
-    - [Logging](#logging)
-- [Documentation](#documentation)
     - [Configuration](#configuration)
         - [Basic Options](#basic-options)
         - [Client Key Options](#client-key-options)
-        - [Advanced Options](#advanced-options)
-            - [Logging](#logging-1)
-            - [Email Verification & Password Reset](#email-verification-and-password-reset)
+        - [Email Verification & Password Reset](#email-verification-and-password-reset)
         - [Using Environment Variables](#using-environment-variables-to-configure-parse-server)
         - [Available Adapters](#available-adapters)
         - [Configuring File Adapters](#configuring-file-adapters)
+        - [Logging](#logging)
+- [Live Queries](#live-queries)
```

Comment:
Seems like it would make more sense for this to be a subsection of `Configuration`?

Reply:
I though so as well. I figured because its a very important feature it needs it own section. Also, its a separate server (you could do startLiveQueryServer: true but that's undocumented).

Reply:
fair point